java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/dialogservice/ReportSavedSessionDialog.java

```diff
@@ -76,21 +76,18 @@ public ReportSavedSessionDialog(final Shell parent,
 
 	private static String getMessage(final IModelSession session,
 			final Messages messages) {
-		final Outcome outcome = session.getValidationResult().getOutcome();
+		final Outcome outcome = session
+				.getValidationsOutcome(ValidationResult::getOutcome);
 		final Path location = session.getToolboxFile().getPath();
 		switch (outcome) {
-		case VALID:
+		case VALID, NOT_SUPPORTED:
```

Comment:
This behavior shouldn't be changed here.
```suggestion
		case VALID:
```

Reply:
you are right